content/resources/lessons/sound-basics/presentation/index.md

```diff
@@ -65,6 +65,17 @@ $ db(V) = 35 dB$
 $ db = -70dB $
 
 </div>
+
+>Notes:
```

Comment:
Should always be one trailing space after a `>`, so `> Notes` not `>Notes`. Applies to all lines in this file